packages/cli/generators/discover/index.js

```diff
@@ -180,6 +191,69 @@ module.exports = class DiscoveryGenerator extends ArtifactGenerator {
     });
   }
 
+  /**
+   * Prompts what naming convention they would like to have for column names.
+   */
+  promptColNamingConvention() {
+    this.namingConvention = {
+      'LoopBack default camelCase convention (Recommend)': false,
+      'keep property names the same as database column names': true,
+    };
+    return this.prompt([
+      {
+        name: 'disableCamelCase',
+        message: `Convert column names to property names:`,
+        type: 'list',
+        choices: Object.keys(this.namingConvention),
+        default: false,
+        when:
+          this.disableCamelCase === undefined &&
+          !this.artifactInfo.disableCamelCase,
+      },
+    ]).then(props => {
+      /* istanbul ignore next */
+      if (!props.disableCamelCase) return;
+      const key = props.disableCamelCase;
+      props.disableCamelCase = this.namingConvention[key];
+
+      Object.assign(this.artifactInfo, props);
+      debug(`props after naming convention prompt: ${props.disableCamelCase}`);
+      return props;
+    });
+  }
+
+  /**
+   * Warns about potential issues for non-LB-default property naming convention.
+   * The prcess can be aborted here.
+   */
+  promptConfirmDisable() {
```

Comment:
a suggestion: maybe display the warning in the previous prompt? Instead of having two prompts?

Reply:
I was thinking that too. But I don't know how to do it :sweat_smile:. Let me try to put them in `validation`.

Reply:
I am still having 2 prompts. And I made some changes, now users could go back to the previous step to choose the column naming convention instead of getting aborted. WDYT?

Reply:
I think it's a bit overkill to have a prompt to confirm the 2nd choice. We can make the warning part of the choice.

Reply:
Yeah I tend to merge the warning to the first prompt. The warning message could be:

> By disabling Camel case, you might need to specify these customized names in relation definition. 

Usually the warning is part of the prompt message, you can use a different color (use `chalk` module) to emphasize and distinguish the warning from the prompt message.

See `chalk`'s example usage in https://github.com/strongloop/generator-loopback/blob/4c9c2068a65aa8a2ee3d47c00efff0b80762fd6c/model/index.js#L107